frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -15,12 +15,18 @@
       <span class="e-subtitle">{{ duration }}</span>
     </td>
     <td style="width: 100%" class="contentrow">
-      <router-link
-        :to="routerLink"
-        class="text-decoration-none text-decoration-hover-underline black--text font-weight-medium d-block"
-      >
-        {{ title }}<br />
-      </router-link>
+      <span style="display: inline-block">
+        <router-link
+          :to="routerLink"
+          class="text-decoration-none text-decoration-hover-underline black--text font-weight-medium d-block"
+        >
+          {{ title }}
+        </router-link>
+      </span>
+      <span style="display: inline-block" class="ml-4">
+        <span style="font-size: 0.7em; color: #aaa">{{ progressLabel }}</span>
```

Comment:
Contrast and font size are extremely low, making it hard for people with vision impairments to read this. How about we use the same color as the .e-subtitle (which is used for the time and location display)?
```suggestion
        <span style="font-size: 0.7em; color: #666">{{ progressLabel }}</span>
```